process/transactionLog/process_test.go

```diff
@@ -0,0 +1,183 @@
+package transactionLog_test
```

Comment:
I prefer to keep the pattern of running tests from a different package, even if in some cases it helps having the same package. Otherwise it will encourage doing the same every time it is hard to test something